include/nuttx/tls.h

```diff
@@ -97,6 +104,7 @@
 
 struct tls_info_s
 {
+  FAR struct task_info_s * tl_task;
```

Comment:
Has https://github.com/apache/incubator-nuttx/blob/97aad21cb1bfc237b67c8e0d9c4988c8ebbbb9f9/include/nuttx/tls.h#L71-L103 been updated to reflect the change?

Reply:
Good point, @no1wudi please fix the comment.(BTW, there are several location, please search and change together).